docs/docs/themes/api-reference.md

```diff
@@ -68,7 +68,13 @@ exports.createPages = async ({ graphql, actions }, themeOptions) => {
 
 ## Component Shadowing
 
-Gatsby Themes allow you to customize any file in a theme's `src` directory by following a file naming convention.
+You can import files from a Gatsby Theme into your project. For example, if you're using `gatsby-theme-tomato`, which has a `Layout` component located at `src/components/layout.js`, you can import it into your poject like this: 
```

Comment:
This should maybe go _after_ the existing snippet? Maybe with a sub-header or something?

Primarily because this is tangential to component shadowing!

Reply:
Got it! Happy to fix.

Reply:
@DSchau is there somewhere better for it to go? I didn't see any existing sections where I thought this fit. Maybe we could create a `using themes` section?

Reply:
That works for me! What say you @ChristopherBiscardi?

Reply:
yeah, let's go with `using components from themes` as a header.

Just noticed we can also get rid of the entire `Add theme transpilation` heading too.

Reply:
Will remove.